src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -166,10 +173,11 @@ private Task ResetLockout(TUser user, CancellationToken token)
                 await UserManager.AccessFailedAsync(user, cancellationToken);
```

Comment:
So in general it looks like we are ignoring any potential mid method failures, like this AccessFailedAsync call actually could fail when updating the user (especially now that we have concurrency built in).  SignInManager currently ignores the failure (this behavior is debateable), but we probably should at least consider logging these results if they are failures, even if we don't fail the higher level sign in method...